harbor/README-v2.11.2.md

```diff
@@ -0,0 +1,16 @@
+# Harbor v2.11.2 Configuration
+
+Below is table that highlights the required fields for the Harbor data values file. 
+
+| Property	                                                        | Value	                             | Description                                                                                                                                                                                                                                                                                                                                   |                                                                                         
+|------------------------------------------------------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|                                                                                        
+| hostname	                                                        | FQDN	                              | The FQDN that you have designated to access the Harbor UI and for referencing the registry in client applications. The domain should be configured in an external DNS server such that it resolves to the Envoy Service IP created by Contour.                                                                                                |
```

Comment:
Mentioned offline, but there's a stray reference to Contour here. Is the hostname required if we're not using Contour (can it be accessed directly by IP)?